pkg/providers/imagefamily/resolver.go

```diff
@@ -221,14 +226,22 @@ func prepareKubeletConfiguration(ctx context.Context, instanceType *cloudprovide
 	return kubeletConfig
 }
 
-func getSupportedImages(familyName *string, kubernetesVersion string) []types.DefaultImageOutput {
+func getSupportedImages(familyName *string, fipsMode *string, kubernetesVersion string) []types.DefaultImageOutput {
 	// TODO: Options aren't used within DefaultImages, so safe to be using nil here. Refactor so we don't actually need to pass in Options for getting DefaultImage.
-	imageFamily := getImageFamily(familyName, kubernetesVersion, nil)
+	imageFamily := getImageFamily(familyName, fipsMode, kubernetesVersion, nil)
+	if lo.FromPtr(fipsMode) == v1beta1.FIPSEnabled {
```

Comment:
"just add a filter parameter" may fit better here too because you don't need to do the filtering here you delegate it to the individual image family provider:
```
return imageFamily.DefaultImages(fipsMode)
```